contracts/assemblyscript/nep21-basic/main.ts

```diff
@@ -108,6 +114,7 @@ export function get_balance(owner_id: AccountId): u128 {
  * receives this information, the allowance may already be changed by the owner.
  * So this method should only be used on the front-end to see the current allowance.
  */
+// do we need this prettier-ignore?
```

Comment:
I would rather remove all of them frankly, I think they clutter the code in the worst possible way but it's annoying to have to keep saving and fix formatting since I prefer consistently formatting method names rather than correctly formatting them

I'll defer to your wisdom here, sir. how should we actually be handling this?

Reply:
> consistently formatting method names rather than correctly formatting

I'd rather let a standard/robot decide and not have to think about it, but I hear what you're saying. But if we have a linter here, then I say we just use it.

Reply:
We don't even have prettier installed in this project, though

If your editor is yelling at you about it, see if you can tell your editor to only yell at you based on the settings in the current repo

Reply:
Correction: it looks like prettier is a dependency of jest, via jest-snapshot, and is therefore in our `yarn.lock` file – maybe this made your editor think it should run some default prettier checks?

`jest-snapshot` probably has this as a dependency by mistake; I've created a PR for it here: https://github.com/facebook/jest/pull/10095